tests/fpga_system_tests/emu/test_emu.py

```diff
@@ -364,13 +381,15 @@ def update_chan(coeff_tuples, offset=0):
 
     # Configure the retimer
     print('Configuring the retimer...')
-    write_tc_reg('retimer_mux_ctrl_1', 0xffff)
-    write_tc_reg('retimer_mux_ctrl_2', 0xffff)
+    write_tc_reg('retimer_mux_ctrl_1', 0b1111111111111111)
+    write_tc_reg('retimer_mux_ctrl_2', 0b1111111111111111)
+    #write_tc_reg('retimer_mux_ctrl_1', 0b0000111111110000)
+    #write_tc_reg('retimer_mux_ctrl_2', 0b1111000000000000)
 
     # Configure the CDR
     print('Configuring the CDR...')
     write_tc_reg('cdr_rstb', 0)
-    write_tc_reg('Kp', 15)
+    write_tc_reg('Kp', 18)
```

Comment:
this didn't appear to make a difference to BER, but it's good to be consistent because I was using 18 everywhere else